.github/workflows/presubmit.yml

```diff
@@ -0,0 +1,157 @@
+name: Presubmits
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
```

Comment:
You currently don't have a mechanism to replicate the `run_if_changed` condition from Prow. To address this, you could add a step at the start of each job using a script or a pre-built action like [dorny/paths-filter@v2 ](https://github.com/dorny/paths-filter) to determine if the job should run based on changed files.